front/src/reducers/osrdsimulation/actions.ts

```diff
@@ -92,14 +87,17 @@ export function updateSelectedProjection(
     });
   };
 }
+
 export function updateSelectedTrainId(selectedTrainId: OsrdSimulationState['selectedTrainId']) {
-  return (dispatch: Dispatch) => {
+  return (dispatch: Dispatch, _: GetState, chartSynchronizer: ChartSynchronizer) => {
+    chartSynchronizer.computePositionValues(selectedTrainId);
```

Comment:
Why do you do that ?

Reply:
this action was recomputing the position values inside the reducer, so now this thunk needs to tell the chart synchronizer to do it.

https://github.com/osrd-project/osrd/pull/6108/files#diff-304c3afa1d4f7a4baead1b8c07ad06812567fab107cd1321dbcf50ca71f98fa1L115

Reply:
Hum okay, I'm not sure it is really useful but fine :) You can close this comment